notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from notebook_utils import download_file
</pre><p>should we add downloadin utils first in case if notebook run not in the repo?</p><p>or you can use hf hub API for downloading https://huggingface.co/docs/huggingface_hub/v0.8.1/en/how-to-downstream</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>